oss_src/unity/python/sframe/data_structures/sarray.py

```diff
@@ -2885,6 +2885,44 @@ def item_length(self):
         with cython_context():
             return SArray(_proxy = self.__proxy__.item_length())
 
+    def random_split(self, fraction, seed=None):
+        """
+        Randomly split the rows of an SArray into two SArrays. The first SArray
+        contains *M* rows, sampled uniformly (without replacement) from the
+        original SArray. *M* is approximately the fraction times the original
+        number of rows. The second SArray contains the remaining rows of the
+        original SArray.
+
+        Parameters
+        ----------
+        fraction : float
+            Approximate fraction of the rows to fetch for the first returned
+            SArray. Must be between 0 and 1.
+
+        seed : int, optional
+            Seed for the random number generator used to split.
+
+        Returns
+        -------
+        out : tuple [SArray]
+            Two new SArrays.
+
+        Examples
+        --------
+        Suppose we have an SArray with 1,024 rows and we want to randomly split
+        it into training and testing datasets with about a 90%/10% split.
+
+        >>> sa = graphlab.SArray(range(1024))
+        >>> sa_train, sa_test = sa.random_split(.9, seed=5)
+        >>> print(len(sa_train), len(sa_test))
+        922 102
+        """
+        from .sframe import SFrame
```

Comment:
I don't think this is going to work either. There is no `.sframe` in GLC. I think you might need to do something like:
https://github.com/dato-code/SFrame/blob/97e13ca91eab82f3a27c65e23316f9617ac0aa99/oss_src/unity/python/sframe/util/__init__.py#L44

`from . import SFrame` I think will work, probably try that one first.